django/utils/cache.py

```diff
@@ -72,8 +72,10 @@ def dictvalue(t):
     # If there's already a max-age header but we're being asked to set a new
     # max-age, use the minimum of the two ages. In practice this happens when
     # a decorator and a piece of middleware both operate on a given view.
-    if 'max-age' in cc and 'max_age' in kwargs:
-        kwargs['max_age'] = min(int(cc['max-age']), kwargs['max_age'])
+    if 'max_age' in kwargs:
```

Comment:
Could you unit test these changes in `test_patch_cache_control`? It's not obvious to me the reason for the changes. Is any update to the comment needed?